src/main/java/uk/ac/cam/cl/dtg/isaac/quiz/IsaacCoordinateValidator.java

```diff
@@ -147,11 +147,22 @@ public final QuestionValidationResponse validateQuestionResponse(final Question
                         String choiceValue = choiceItem.getCoordinates().get(dimensionIndex);
                         String submittedValue = submittedItem.getCoordinates().get(dimensionIndex);
 
-                        Integer sigFigs = ValidationUtils.numberOfSignificantFiguresToValidateWith(submittedValue,
-                                significantFiguresMin, significantFiguresMax, log);
-
-                        boolean valuesMatch = ValidationUtils.numericValuesMatch(choiceValue, submittedValue, sigFigs, log);
+                        boolean valuesMatch = false;
 
+                        if (submittedValue.isEmpty()) {
+                            feedback = new Content(FEEDBACK_INCOMPLETE_ANSWER);
+                        }
+                        else {
```

Comment:
Can this be consistent with the rest of the codebase as `} else {` on a single line? 🙏

Reply:
```suggestion
                        } else {
```